app/scripts/views/settings/change_password.js

```diff
@@ -45,6 +45,7 @@ define(function (require, exports, module) {
           self.relier
         )
         .then(function () {
+          self.logEvent('settings.change_password.success');
```

Comment:
Also I think you should be able to use `logViewEvent` and just log `'success'`

Ref: https://github.com/mozilla/fxa-content-server/blob/issue-3396-measure-password-change/app/scripts/views/settings/delete_account.js#L45